schedule/yast/btrfs/btrfs_opensuse.yaml

```diff
@@ -2,7 +2,6 @@ name:           btrfs
 description:    >
   Validate default installation with btrfs.
 vars:
-  FILESYSTEM: btrfs
```

Comment:
we still cannot remove it here, right? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4357ef1a9764fa73e8074c02dd2019bd15fc1957/lib/y2_base.pm#L89

Reply:
good catch ! I'll have a look.

Reply:
I'm not so sure about that one. Collecting those btrfs-specific logs can be useful, but in this form is not really useful. If we want to keep it, it makes sense to completely revert the commit and keep that variable.

Reply:
I kept the variable here in the end.

Reply:
It looks like this variable is only used in this test suite.